src/main/java/com/coherent/training/selenium/stanila/pages/DynDataLoadingPOM.java

```diff
@@ -0,0 +1,28 @@
+package com.coherent.training.selenium.stanila.pages;
+
+import com.coherent.training.selenium.stanila.base.BaseClass;
+import lombok.SneakyThrows;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class DynDataLoadingPOM extends BaseClass {
```

Comment:
do not extend the pom class from base test class.
define BasePOM class

Reply:
Implemented